lib/event_store_http_client/events/write.rb

```diff
@@ -14,14 +14,12 @@ class Write
         dependency :settings
         dependency :client
 
-        def initialize(type, data, stream_name, version)
```

Comment:
Well, it will generally be writing to "account-621" or similar.  And *THAT* knowledge will most likely not be available until write-time.